client/src/components/PrintButton.tsx

```diff
@@ -18,8 +18,17 @@ const PrintButton = ({
   ...rest
 }: React.PropsWithChildren<PrintButtonProps>) => {
   const [isPrinting, setIsPrinting] = useState(false)
+  const [showPrintingError, setShowPrintingError] = useState(false)
```

Comment:
nit: Why not be more specific?
```suggestion
  const [isConnectedError, setIsConnectedError] = useState(false)
```
or with this suggestion one you'd have to flip the boolean logic around…

```suggestion
  const [isConnected, setIsConnected] = useState(true)
```

Reply:
punting on this one cause a nit and need to get these merged ASAP.